src/reducers/app.js

```diff
@@ -0,0 +1,18 @@
+import { MENU_SATE_CHANGE, REDIRECT_FROM_PAGE } from '../action/HeaderActions'
+
+const initialState = () => {
+    return {
+
+    }
+}
+
+
+export function appReducer(state = initialState(), action) {
+
+    switch (action.type) {
+
+

```

Comment:
Don't leave empty lines. It isn't a good idea. It will be better if you will leave comments(I wrote before).